public/app/plugins/datasource/influxdb/query_builder.ts

```diff
@@ -43,6 +43,11 @@ export class InfluxQueryBuilder {
     } else if (type === 'MEASUREMENTS') {
       query = 'SHOW MEASUREMENTS';
       if (withMeasurementFilter) {
+        if (withMeasurementFilter.match('/')) {
```

Comment:
not sure I understand this condition, looks like withMeasurementFilter always needs full regex escaping here.  
Where have a utility lib (kbn) that has a regexEscape function 
kbn.regexEscape(value)

Reply:
That's a better solution. @SamuelToh could you update the PR to use that function instead?

Reply:
Applied review recommendations. Please re-review them. @torkelo  @bergquist

Reply:
@torkelo I think you did a mistake here. We have to excape the '/' so that it is not recognised as the end of regex. But with kbn.regexEscape all characters like *.?+ are escaped and that disables the regex.
Currently it is not possible to filter measurements by regex.
His first attempt was perfectly fine.

Cheers,
Chris